snapcraft/cli/__init__.py

```diff
@@ -0,0 +1,103 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import logging
+
+import click
+
+import snapcraft.internal.dirs
+from snapcraft.internal import deprecations
+from snapcraft.internal import log
+from . import echo
+from .assertions import assertionscli
+from .discovery import discoverycli
+from .lifecycle import lifecyclecli
+from .store import storecli
+from .parts import partscli
+from .help import helpcli
+from .ci import cicli
```

Comment:
Because @kyrofa and I like them more :-P

Really means we can move this package around (or others) with less effort later.